<?xml version="1.0" ?>
<launch>
  <!-- params -->
  <param name="forest_file" value="$(find leg_tracker)/config/trained_leg_detector_res=0.33.yaml" />
  <param name="scan_topic" value="/right_scan" />
  <param name="fixed_frame" value="odom"/>
  <param name="scan_frequency" value="7.5"/>  
  <param name="max_detect_distance" value="4.0"/>
  <param name="robot_description" command="xacro --inorder '$(find leg_tracker)/urdf/FRMI_model.urdf.xacro'" />

  
  
  <!-- run detect_leg_clusters -->  
  <node pkg="leg_tracker" type="detect_leg_clusters" name="detect_leg_clusters" output="screen"/>

  <!-- run joint_leg_tracker -->
  <node pkg="leg_tracker" type="joint_leg_tracker.py" name="joint_leg_tracker" output="screen"/>
 
   <!-- run local_occupancy_grid_mapping -->
  <node pkg="leg_tracker" type="local_occupancy_grid_mapping" name="local_occupancy_grid_mapping" output="screen"/>
  
  <!-- run rviz -->                
  <node pkg="rviz" type="rviz" name="rviz"
    args="-d $(find leg_tracker)/rosbags/demos/rviz/demo_moving_super_crowded.rviz"/>  

  <!-- play rosbag -->  
  <node pkg="rosbag" type="play" name="player" output="screen" required="true" 
    args="--delay=4
    --clock 
    --rate=1.0
    -s 0
    $(find leg_tracker)/rosbags/demos/demo_moving_super_crowded.bag"/>
  <param name="use_sim_time" value="true" />
    
</launch>

